WordPress/Classes/ViewRelated/Me/MyProfileViewController.swift

```diff
@@ -2,60 +2,88 @@ import UIKit
 import RxSwift
 import WordPressShared
 
-class MyProfileController: NSObject {
+func MyProfileViewController(account account: WPAccount) -> ImmuTableViewController {
```

Comment:
Ooh this is sneaky! It looks like an init method but it isn't... Not sure I like it but I don't not like it either.